cosmos-server/src/main/scala/com/mesosphere/cosmos/storage/LocalObjectStorage.scala

```
@@ -117,6 +137,16 @@ final class LocalObjectStorage private(storageDir: Path, scratchDir: Path, stats
 
   override def getUrl(name: String): Option[Uri] = None
 
+  override def getCreationTime(name: String): Future[Option[Instant]] = {
+    pool {
+      val absolutePath = storageDir.resolve(name)
+      val attr = Try {
+        Files.readAttributes(absolutePath, classOf[BasicFileAttributes])
+      }
```

Really it's the `toOption` below that's hiding the exceptions...